core/loadpoint.go

```diff
@@ -634,8 +628,7 @@ func (lp *Loadpoint) Prepare(uiChan chan<- util.Param, pushChan chan<- push.Even
 
 	// read initial charger state to prevent immediately disabling charger
 	if enabled, err := lp.charger.Enabled(); err == nil {
-		if lp.enabled = enabled; enabled {
-			lp.guardUpdated = lp.clock.Now()
+		if lp.setEnabled(enabled, true); enabled {
```

Comment:
```suggestion
		if lp.setEnabled(enabled, false); enabled {
```

Hier wird nichts geschaltet, insofern sollte das egtl. false sein (schon immer).

Reply:
Ich denke der Gedanke daran war, dass der guardTimer beim Start erst einmal zu laufen beginnt. Ich wollte offen gestanden nicht zu viel auf einmal anfassen, deswegen habe ich es drin gelassen. Stimme Dir aber zu - insofern weg damit!